drivers/hmis/app/models/hmis/hud/custom_assessment.rb

```diff
@@ -147,7 +147,7 @@ def save_submitted_assessment!(current_user:, as_wip: false)
         enrollment.save!
         enrollment.touch # Update even if no changes to Enrollment
         # Move Enrollment out of WIP if this is a submitted intake
-        enrollment.save_not_in_progress if intake?
+        enrollment.save_not_in_progress! if intake?
```

Comment:
I originally aliased it because I made the change to the function signature, and didn't want to break code in any places where we were calling it. However I am fairly confident that I updated the code everywhere, so I'm going to remove the alias. `!` is more clear here because it hints at the fact that this function does call `save!`